duckduckgo/duckduckgo-privacy-extension
# Enable browsingTopics protection

**Reviewer:** @jonathanKingston, @sammacbeth 



## Description:

https://app.asana.com/0/1163321984198618/1201719300245228/f

`browsingTopics` is effectively replacing `floc`. This PR turns our `floc` protection into a `browsingTopics` protection.


## Steps to test this PR:
Browsing topics is not yet deployed in current versions of chrome and we're basing this protection on currently available documentation ([see here](https://blog.google/products/chrome/get-know-new-topics-api-privacy-sandbox/)). The content script can be verified by adding a log line to ensure the script is properly injected.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
